package.json

```diff
@@ -32,10 +34,12 @@
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-cypress": "^2.11.3",
     "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsdoc": "^36.0.6",
     "fs-extra": "^10.0.0",
-    "html-validate": "^5.2.0",
```

Comment:
Is there any reason why you downgraded the html-validate version? This upgrade was made through the dependabot and checked it worked properly.

Reply:
Truth is I didn't paid attention to it.
I'm bumping the version to the latest.